docs(pair-programming): Require issue or PR and remove getting started sessions (#20465)
gatsbyjs/gatsby: docs/contributing/pair-programming.md

## How community pair programming sessions work

> 1.  Select or create [an issue](https://github.com/gatsbyjs/gatsby/issues) you would like to work on and work on it yourself.

Reviewer: ```suggestion
1.  Select or create [an issue](https://github.com/gatsbyjs/gatsby/issues) you would like to work on and work on it yourself
```

2.  Create a [Pull Request](https://github.com/gatsbyjs/gatsby/compare) when you encounter any problem.
3.  Sign up for any available slot on [the open pairing calendar][cal]

> 4.  You’ll be paired with one of the Gatsby Inkteam

Reviewer: Maybe it's not obvious what Inkteam means?

```suggestion
4.  You’ll be paired with a Gatsby team member
```

5.  Join the video meeting during your time slot (we are using Zoom)
6.  We'll help you with your PR or Issue!

### What we expect in pair programming sessions

These sessions are intended for people who would like to contribute to Gatsby. We will pair program together to solve an issue or pull request related to Gatsby.

If you're interested in support for a commercial Gatsby project, please [get in touch via the contact page](https://www.gatsbyjs.com/contact-us/).

We also expect the following from pair programming participants:

> - These sessions tend to work best when you have a specific goal for the session. Preferably you have already selected and worked on an issue and have created a Pull Request with your work.

Reviewer 2: ```suggestion
- These sessions tend to work best when you have a specific goal for the session. Preferably you have already selected and worked on an issue or have created a pull request with your work.
```

`Or` over `and` since some folks might've picked an issue but would like to start on the call?

> - These sessions tend to work best when you have a specific goal for the session. Preferably you have already selected an issue or have started work and created a work-in-progress pull request.

Reviewer: I think we are making it a requirement (not optional) that someone chooses an issue or PR?

```suggestion
- These sessions work best when you have a specific goal for the session. You should choose an issue or pull request that you'd like to work on.
```
